lib/ansible/modules/windows/win_service.py

```diff
@@ -113,6 +113,20 @@
     - A newly created service will default to C(LocalSystem).
     type: str
     version_added: '2.3'
+  recovery_reset_interval:
+    description:
+      - Reset failure counter after this number of seconds.
+    type: int
+    default: 0
+    version_added: '2.10'
+  recovery_actions:
+    description:
```

Comment:
We should add a note by saying win_service will run idempotency checks on the actions as a whole and not as individual entries. This is because the list order is important to determine the order of the actions themselves. Hopefully this makes sense?

Also what happens when someone specifies an empty list. In my mind this should just remove the recovery actions from the service but the code seems like it expects 3 or fails when not null.